src/app/inventory/actions.ts

```diff
@@ -62,6 +62,7 @@ export const itemMoved = createAction('inventory/MOVE_ITEM')<{
  * We need to update the inventory with the updated item and any removed/added items.
  */
 export const awaItemChanged = createAction('inventory/AWA_CHANGE')<{
+  item: DimItem | null;
```

Comment:
I had to start passing this here instead of constructing it in the reducer because the reducer called makeItem which later grabbed things directly out of state (custom stat totals setting) which is forbidden in a reducer.